spec/asyncapi.md

```diff
@@ -388,7 +391,10 @@ A single server would be described as:
   "url": "development.gigantic-server.com",
   "description": "Development server",
   "protocol": "kafka",
-  "protocolVersion": "1.0.0"
+  "protocolVersion": "1.0.0",
+  "traits": [
+    { "$ref": "#/components/serverTraits/kafka" }
+  ]
```

Comment:
Would it make sense to create another example for traits? I find it confusing here.

Reply:
TBH I don't understand. What you wanna add here? Similar examples of using traits we have in message and operation traits with `$ref`.

Reply:
Yeah, I know, it's just that by looking at this example I simply would not understand what a server trait is if it were my first time with the spec.